### ISSUE TITLE
Standard metas are not discoverable

### ISSUE THREAD

**Issue by octomarat** [JetBrains/intellij-arend#261]
If you don't know about standard metas, it is very hard to discover/browse them. If you know the name of some meta, you can import it, then go to its declaration. This allows you read the docs and browse metas declared in the same file. But I believe the plugin can do better. Specifically:

1. Show all the standard metas in External Libraries as Arend files. Plugin actually show them as Java classes in `arend-lib/ext/org.arend.lib/StdExtension`.
2. "Navigate | Symbol" and "Navigate | File" should show corresponding metas and files with metas.

**Comment by octomarat** [comment]
Looks similar to #254.

**Comment by ice1000** [comment]
I tried to document 'em: https://arend-lang.github.io/documentation/standard-tactics but I didn't finish

**Comment by octomarat** [comment]
Yeah, I know about that page and it is really helpful. Especially the examples. So, thank you for the efforts!

**Comment by ice1000** [comment]
You could improve that
